CONTRIBUTING.md

```diff
@@ -1,20 +1,28 @@
 # Contributing
 
 - Creating a bundle: `npm prepublish`
-- Working on docs: `npm run docs:watch`
-- Developing with tests: `npm run test:watch`
+- Working on docs: `npm run docs:watch` / `yarn docs:watch`
+- Developing with tests: `npm run test:watch` / `yarn test:watch`
+- Fixing js and ts code style: `npm run style:fix` / `yarn style:fix`
+- Commiting (with commitizen): `npm run commit` / `yarn commit`
 
 ## Committing
 
-Ensure you have [commitizen](https://github.com/commitizen/cz-cli) installed and commit using `git cz`.
+We are using semantic-release and conventional-changelog for releasing, so our commit messages have to follow strict rules.
+
+The commit message formatting can be added using a typical git workflow or through the use of a CLI wizard ([Commitizen](https://github.com/commitizen/cz-cli)).
+
+To use the wizard, run `npm run commit` or `yarn commit` in your terminal after staging your changes in git.
```

Comment:
Isn't this just an alias for `git cz`? Not sure if we need the indirection of subverting the standard command.